Sources/Extensions/CollectionViewManager+UICollectionViewDataSource.swift

```diff
@@ -8,15 +8,15 @@ import UIKit.UICollectionView
 
 extension CollectionViewManager: UICollectionViewDataSource {
     
-    open func collectionView(_ collectionView: UICollectionView,
-                             numberOfItemsInSection section: Int) -> Int {
+    open func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         return _sectionItems[section].cellItems.count
     }
     
-    open func collectionView(_ collectionView: UICollectionView,
-                             cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    open func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cellItem = self.cellItem(for: indexPath)!
```

Comment:
I know it's been here for a long time, but I think it's time to get rid of this force-cast.

Reply:
Actually I have no idea how to rid it of in elegant way. The only possible solution I see is add `fatalError` if there are no corresponding cell item. 🤔